wire-runtime/src/main/java/com/squareup/wire/Message.java

```diff
@@ -151,7 +148,7 @@ public Builder() {
      * Returns a byte string with this message's unknown fields. Returns an empty byte string if
      * this message has no unknown fields.
      */
-    public ByteString buildUnknownFields() {
+    public final ByteString buildUnknownFields() {
```

Comment:
I like final on types wherever we can, and if we can’t final on methods is a good fallback